sources/packages/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -140,10 +140,11 @@ export class NoteAESTController extends BaseController {
     if (!student) {
       throw new NotFoundException("Student not found.");
     }
-    const studentNote = transformToNoteEntity(payload, userToken.userId);
```

Comment:
for my info, why `this transformToNoteEntity` is removed

Reply:
The transformToNoteEntity utility creates note entity model for the sake of saveStudentNote  service method which is deprecated and has been replaced.